src/main/kotlin/blackjack/domain/state/notstarted/NotStarted.kt

```diff
@@ -0,0 +1,37 @@
+package blackjack.domain.state.notstarted
+
+import blackjack.domain.card.Card
+import blackjack.domain.state.State
+import java.math.BigDecimal
+
+class NotStarted : State {
+
+    override val isRunning: Boolean
+        get() = false
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    override val isRunning: Boolean = false
```